projects/angular/components/ui-tree-select/src/ui-tree-select.component.scss

```
@@ -0,0 +1,48 @@
+.virtual-scroll-container {
+  border: 2px solid cyan;
+  height: 500px;
+  overflow: auto;
+}
+
+// .node {
+//   display: block;
+//   height: 24px;
+// }
+
+.tree-item-container {
+  &:active,
+  &:focus,
+  &:focus-visible,
+  &:focus-within {
+    background-color: silver;
+  }
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 100%;
+
+  &.selected {
+    background-color: gold;
+  }
+}
+
+@mixin perLevelPadding($level, $padding) {
+  .node-level-#{$level} {
+
+    &:not(.node-type-leaf) {
+      padding-left: max(0, calc($padding * ($level)));
```

Can you ever have negative levels? As for the padding I think you there's only negative margin.
```suggestion
      padding-left: calc($padding * ($level));
```

---

fair point.
took note of the feedback and moved the styles to the playground.